modules/consensus/synchronize.go

```diff
@@ -10,11 +10,16 @@ import (
 	"github.com/NebulousLabs/bolt"
 )
 
-const (
+var (
 	// MaxCatchUpBlocks is the maxiumum number of blocks that can be given to
 	// the consensus set in a single iteration during the initial blockchain
 	// download.
-	MaxCatchUpBlocks = 10
+	MaxCatchUpBlocks = func() types.BlockHeight {
+		if build.Release == "testing" {
+			return 3
+		}
+		return 10
```

Comment:
Being explicit prevents mistakes, and makes it easier to tweak parameters if you are doing debugging or specific testing. It's a lot easier to memorize/read/lookup if you know that the definitions are consistent throughout the file, rather than trying to remember if only testing or only dev is special for this or that constant.